site/plugins.json

```diff
@@ -1008,7 +1008,8 @@
     "package": "@chiselstrike/netlify-plugin",
     "description": "Sync Netlify sites and ChiselStrike builds from code in the same Git repository.",
     "repo": "https://github.com/chiselstrike/netlify-plugin",
-    "version": "0.1.0"
+    "version": "0.1.0",
+    "status": "DEACTIVATED"
```

Comment:
```
> npm info @chiselstrike/netlify-plugin
npm error code E404
npm error 404 Not Found - GET https://registry.npmjs.org/@chiselstrike%2fnetlify-plugin - Not found
npm error 404
npm error 404  '@chiselstrike/netlify-plugin@*' is not in this registry.
```

This npm package was either removed or made private. Either way it's no longer installable. Github repo is still up, but there was no movement there in 3 years